### ISSUE TITLE
Typescript interface doesn't allow the usage of a closure for a "class" attribute

### ISSUE THREAD

**Issue by raydf** [luwes/sinuous#122]
The next code snippet doesn't work:

```js
const dropdownClassesPrefix = "origin-top-right absolute right-0 mt-1 w-48 rounded-lg shadow-lg transition "

...

<div class={() => dropdownClassesPrefix + " ease-out duration-100 transform opacity-100 scale-100"}></div>

```

This is related to the closed issue:

https://github.com/luwes/sinuous/issues/40

I believe the typescript definition only allows string in the "class" attribute but it should handle an observable closure function. Without this, one cannot make changes dynamically to the css classes, based on state.

**Comment by lynlevenick** [comment]
I don't see this locally since the fix for #105; are you running a version below 0.27.1?

**Comment by raydf** [comment]
@lynlevenick thanks for the response. You're right, i was using an old version of sinuous.
